### PR TITLE
feat: Typography supports the prompt to turn off the copy button

### PR DESCRIPTION
**Pull request by zzz111**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

🐛[BUG] pro-table 的 ellipsis = true， 会出现多个tooltip
https://github.com/ant-design/pro-components/issues/29

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |Typography supports the prompt to turn off the copy button|
| 🇨🇳 Chinese |Typography支持关闭复制按钮的提示|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/demo/interactive.md](https://github.com/ant-design/ant-design/blob/fixTooltip/components/typography/demo/interactive.md)
[View rendered components/typography/index.en-US.md](https://github.com/ant-design/ant-design/blob/fixTooltip/components/typography/index.en-US.md)
[View rendered components/typography/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/fixTooltip/components/typography/index.zh-CN.md)

### CODE REVIEW

**Comment by afc163** [comment]
`tooltips: ['', '']` 这样就能关。

**Comment by chenshuai2144** [comment]
image


就叫 Typography ，不应该改成排版的

**Comment by zzz111** [comment]
> `tooltips: ['', '']` 这样就能关。

试了一下，这样关不了，有默认值

**Comment by afc163** [comment]
> 试了一下，这样关不了，有默认值

可以修正一下，这样预期应该可以关。

**Comment by chenshuai2144** [comment]
tooltips: ['', ''] 有点丑，还是 false 感觉优雅一点

**Comment by afc163** [comment]
`tooltips: []` 你是可以选择只关一个的。

**Comment by chenshuai2144** [comment]
tooltips:[false,false] 这样呢

**Comment by afc163** [comment]
可以的，关键不是 false，是要把 tooltips 的 falsy 值都传给 Tooltip。

File: components/typography/Base.tsx [hunk 1]
```diff
@@ -380,21 +380,28 @@ class Base extends React.Component<InternalBlockProps, BaseState> {
 
     const prefixCls = this.getPrefixCls();
 
-    const title = copied
-      ? (copyable as CopyConfig).tooltips?.[1] || this.copiedStr
```
Comment (afc163) [inline components/typography/Base.tsx:384]: tooltips 存在时始终走 tooltips。

**Comment by llwslc** [comment]
这个功能已经合并到feature, 你看下这个. https://github.com/ant-design/ant-design/pull/25953

**Pull request closed without merging**